loopchain/channel/channel_service.py

```diff
@@ -434,7 +432,44 @@ def __init_node_subscriber(self):
 
         return score_info
 
+    def _load_peers_from_iiss(self):
+        utils.logger.debug(f"load peers from iiss...")
+
+        request = {
+            "method": "ise_getPReps"
+        }
+
+        request = convert_params(request, ParamType.call)
+        stub = StubCollection().icon_score_stubs[ChannelProperty().name]
+        response = stub.sync_task().call(request)
+        response_to_json_query(response)
+
+        utils.logger.spam(f"from icon service channels is {response}")
+
+        peer_ids = ''
+        for preps in response["result"]["preps"]:
+            peer_ids += preps['id']
+
+        if self.__peer_manager.get_peer_ids_hash(peer_ids) == self.__peer_manager.peer_ids_hash():
+            utils.logger.debug(f"There is no change in peers.")
+            return
+
+        utils.logger.debug(f"Peer manager have to update with new list.")
+        self.__peer_manager.reset_peers(check_status=False)
+
+        order = 1
+        for rep_info in response["result"]["preps"]:
```

Comment:
why don't you using with `enumerate` as following?

```suggestion
        for order, rep_info in enumerate(response["result"]["preps"], 1):
```